regression/syscall_test/blocklists/creat_test

```diff
@@ -0,0 +1 @@
+CreatTest.CreatTruncatesExistingFile
```

Comment:
This test failed because of the truncation failure. And we do not support `sys_truncate` now.